recipe/emailverification/emailverificationClaim.go

```diff
@@ -37,9 +37,14 @@ func NewEmailVerificationClaim() (*claims.TypeSessionClaim, evclaims.TypeEmailVe
 	evClaim, booleanClaimValidators := claims.BooleanClaim("st-ev", fetchValue, nil)
 
 	getLastRefetchTime := func(payload map[string]interface{}, userContext supertokens.UserContext) *int64 {
-		if value, ok := evClaim.GetValueFromPayload(payload, userContext).(map[string]interface{}); ok {
-			val := value["t"].(int64)
-			return &val
+		if value, ok := payload[evClaim.Key].(map[string]interface{}); ok {
+			switch t := value["t"].(type) {
+			case int64:
+				return &t
+			case float64:
+				it := int64(t)
+				return &it
```

Comment:
why would it be one or the other? You are setting it, so shouldn't it always be what you set it?

Reply:
when it gets converted to JSON and then back, it becomes a float64